mcproto/utils/abc.py

```diff
@@ -69,6 +69,14 @@ class Serializable(ABC):
 
     Any class that inherits from this class and adds parameters should use the :func:`~mcproto.utils.abc.define`
     decorator.
+
+    Example usage:
+
+    .. literalinclude:: /../tests/mcproto/utils/test_serializable.py
+        :start-after: # region ToyClass
+        :end-before: # endregion
+        :linenos:
+        :language: python
     """
 
     __slots__ = ()
```

Comment:
Consider removing in favor of attrs converters & validators, as discussed on [discord](https://discord.com/channels/936788458939224094/945384318660919306/1252668384739594282)

Also applies to the use of the `validate` method, which can probably be completely removed now.